package.json

```diff
@@ -1,7 +1,7 @@
 {
   "homepage": "http://valodyapapikyan.github.io/ameria_ui_library",
   "name": "dino_ui_tools",
-  "version": "1.2.144",
+  "version": "1.2.145",
```

Comment:
I don't know why, but if you check latest version of this package in npm, it's "1.2.145". So the publish will not work. You need to increase the version.